Makefile

```diff
@@ -7,33 +7,58 @@
 # Phony Targets
 .PHONY := build verify_app_target_tuple_config verify_app_target_tuple_is_specified clean all
 
-# Environment Options
-UWRT_FIRMWARE_MAX_JOBS ?= $(nproc)
+# Options
+CLEAN ?=
+BUILD_PROFILE ?= develop
```

Comment:
only profile supported rn is develop... because of upstream mbed-tools issues. working on resolving but develop is good enough for most our purposes anyways